subgraphs/uniswap-forks/protocols/mm-finance/config/networks/cronos/cronos.json

```diff
@@ -0,0 +1,13 @@
+{
+  "deployment": "MM_FINANCE_CRONOS",
+  "network": "mainnet",
```

Comment:
I copied this value from vvs-finance, is it correct? Or should it be `cronos` or something? @steegecs

Reply:
Yes - This is fine. This is how I have named this parameter in the uniswap forks. it is used for configuring the constants.

Reply:
<img width="348" alt="Screen Shot 2022-08-06 at 6 21 33 PM" src="https://user-images.githubusercontent.com/56660047/183267740-c1b70ffd-cf56-4f81-b5c3-71b61f2c3163.png">

Should be the same as the value in the deploy.ts file in the configurations folder.